common/lib/api-client/index.js

```diff
@@ -17,6 +17,8 @@ module.exports = function() {
     logger: IS_DEV,
   })
 
+  instance.pluralize.addUncountableRule('cancel')
```

Comment:
I feel this is rogue enough that it probably warrants some kind of comment as to why it's here.

And possibly even within the model itself as well to explain the direct relationship...?